### DISCUSSION TITLE
osu!lazer can't log in

### DISCUSSION THREAD

**Discussion by flippette** [ppy/osu#13240] [Q&A] [answered]
**Describe the bug:** 
When I hit fill the username and password fields and hit "Log in", after 2-5 seconds it puts me back at the login menu.
**Screenshots or videos showing encountered issue:** 

https://user-images.githubusercontent.com/43138244/120074179-d6d05580-c0c5-11eb-9888-00ca08cd2f1a.mp4

**osu!lazer version:** 
master (f144fcf)
**Logs:**
[network.log](https://github.com/ppy/osu/files/6564800/network.log)
[performance.log](https://github.com/ppy/osu/files/6564801/performance.log)
[runtime.log](https://github.com/ppy/osu/files/6564802/runtime.log)
[database.log](https://github.com/ppy/osu/files/6564803/database.log)

**Comment by enoslayd** [comment]
If your running on debug, you need to create an account first since it connects to the dev server

**Comment by flippette** [comment]
> If your running on debug, you need to create an account first since it connects to the dev server

I'm running on release.

**Comment by peppy** [comment]
Please visit the osu! site and fill in the cloudflare captcha. Your IP address is blacklisted.

**Comment by flippette** [comment]
After filling the captcha, it still can't log in.

**Comment by frenzibyte** [comment]
I've converted this into a Q&A discussion as that's the place where such issues should reside in, as you would see here if you were to submit an issue:
![CleanShot 2021-05-29 at 19 54 42](https://user-images.githubusercontent.com/22781491/120078340-b8fbf400-c0b7-11eb-9113-dded18d6ea1f.png)

---

As for your issue, can you please attach the logs again now that you've filled the captcha.

**Reply by flippette** [reply]
Sure thing, here they are.
[network.log](https://github.com/ppy/osu/files/6565037/network.log)
[performance.log](https://github.com/ppy/osu/files/6565038/performance.log)
[runtime.log](https://github.com/ppy/osu/files/6565039/runtime.log)
[database.log](https://github.com/ppy/osu/files/6565040/database.log)

**Comment by flippette** [comment, marked as answer]
Additionally, I also can't install osu!stable. Only the website works.

**Reply by gagahpangeran** [reply]
Probably it's the same issue, your IP is blocked by cloudflare. Try restart your router to get new IP address (I assume it's using dynamic IP). If the problem still occur, try contact your ISP.

**Reply by flippette** [reply]
It is the same issue, though I don't have a dynamic IP. I reconnected my router to the WAN network and the ISP gave me an IP that actually works. osu!lazer also works now.
